.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,16 @@
+on:
+  push:
+    branches:
+      - main
+
+name: "Trigger: Push action"
+permissions: {}
+
+jobs:
+  shellcheck:
+    name: Shellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run ShellCheck
+        uses: ludeeus/action-shellcheck@master
```

Comment:
Can we use a specific release? It is not recommended to use latest commit on GHA both for reproducibility and security reasons.

Reply:
This is a linter, so aside from the security concerns, you'd probably want a recent release anyhow.  I'll pin to latest released version as of time of this post.

Reply:
> This is a linter, so aside from the security concerns, you'd probably want a recent release anyhow.

This is a stable linter, so sure. In Python land I've seen, more than I would like to admit, bugs due to folks blindly following the latest linter advice.